src/Maestro/tests/Scenarios/azdoflow-batched.ps1

```diff
@@ -52,13 +52,13 @@ try {
     $targetRepoUri = Get-AzDO-RepoUri $targetRepoName
 
     Write-Host "Creating a test channel '$testChannelName'"
-    Darc-Add-Channel $testChannelName "test"
+    Darc-Add-Channel -channelName $testChannelName -classifcation "test"
```

Comment:
The classification param didn't go through in this test.

`Option 'c, classification' has no value.`

Reply:
AAAAAAA